spec/system/visit_user_journey_check_your_certificate_page_spec.rb

```diff
@@ -30,16 +36,69 @@
-      expect(page).to have_content 'VSP'
+      expect(page).to have_content 'SP'
       expect(page).to have_content 'Encryption'
       click_button 'Use this certificate'
       expect(current_path).to eql confirmation_path(sp_component.component_type, sp_component.id, sp_component.encryption_certificate_id)
     end
   end
+
+  context 'signing journey' do
+    it 'expect msa component, signing certificate and successfully goes to next page' do
+      msa_signing_certificate = create(:msa_signing_certificate)
+      msa_component = msa_signing_certificate.component
+      visit upload_certificate_path(msa_component.component_type, msa_component.id, msa_component.signing_certificates[0])
+      fill_in 'certificate_value', with: msa_signing_certificate.value
+      click_button 'Continue'
+      expect(current_path).to eql check_your_certificate_path(msa_component.component_type, msa_component.id, msa_component.signing_certificates[0])
+      expect(page).to have_content 'MSA'
+      expect(page).to have_content 'Signing'
+      click_button 'Use this certificate'
+      expect(current_path).to eql confirmation_path(msa_component.component_type, msa_component.id, msa_component.signing_certificates[0])
+    end
+
+    it 'expect vsp component, signing certificate and successfully goes to next page' do
+      vsp_signing_certificate = create(:vsp_signing_certificate)
+      vsp_component = vsp_signing_certificate.component
+      visit upload_certificate_path(vsp_component.component_type, vsp_component.id, vsp_component.signing_certificates[0])
+      fill_in 'certificate_value', with: vsp_signing_certificate.value
+      click_button 'Continue'
+      expect(current_path).to eql check_your_certificate_path(vsp_component.component_type, vsp_component.id, vsp_component.signing_certificates[0])
+      expect(page).to have_content 'VSP'
+      expect(page).to have_content 'Signing'
+      click_button 'Use this certificate'
+      expect(current_path).to eql confirmation_path(vsp_component.component_type, vsp_component.id, vsp_component.signing_certificates[0])
+    end
+  
+    it 'expect sp component, signing certificate and successfully goes to next page' do
+      sp_signing_certificate = create(:sp_signing_certificate)
+      sp_component = sp_signing_certificate.component
+      visit upload_certificate_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
+      fill_in 'certificate_value', with: sp_signing_certificate.value
+      click_button 'Continue'
+      expect(current_path).to eql check_your_certificate_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
+      expect(page).to have_content 'SP'
+      expect(page).to have_content 'Signing'
+      click_button 'Use this certificate'
+      expect(current_path).to eql confirmation_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
```

Comment:
```suggestion
      expect(current_path).to eql confirmation_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates.first)
```

Syntactic sugar. Non blocking